src/containers/Search/Search.tsx

```
@@ -31,6 +32,7 @@ export interface SearchProps {
   searchParam?: any;
   setState?: any;
   searchId?: string | null;
+  setSearchParam?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)